src/main/java/com/fakng/fakngagrgtr/parser/spotify/spotify-sample.json

```diff
@@ -0,0 +1,53 @@
+{
+  "main_categories": [
+    {
+      "name": "Engineering",
+      "slug": "engineering"
+    },
+    {
+      "name": "Data, Research & Insights",
+      "slug": "data-research-insights"
+    },
+    {
+      "name": "Product",
+      "slug": "product"
+    },
+    {
+      "name": "Workplace Services",
+      "slug": "workplace-services"
+    },
+    {
+      "name": "Sales",
+      "slug": "sales"
+    }
+  ],
+  "result": [
+    {
+      "id": "director-of-engineering-experience-2",
+      "text": "Director of Engineering, Experience",
```

Comment:
Can we get some more data? We need the location date, and description of the position, etc
Is there a way to fetch some more?